code/modules/client/preferences.dm

```diff
@@ -1125,6 +1125,9 @@ datum/preferences
 */
 						for(var/L in all_languages)
 							var/datum/language/lang = all_languages[L]
+							if(lang.name == "an unknown language")

```

Comment:
Checking the language's name probably isn't the best way to do it, that said, for whatever reason languages use a global list to store every single language which is a bit strange.